packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx

```diff
@@ -138,9 +138,9 @@ export const menuTabs: ForumOptions<Array<MenuTab>> = {
       showOnCompressed: true,
     // next 3 are subItems
     }, {
-      id: 'leastwrong',
+      id: 'bestoflesswrong',
       title: 'Best of LessWrong',
-      link: '/leastwrong',
```

Comment:
We have linked to /leastwrong a bunch. I think we want to make sure to have a redirect for the old URL.

Reply:
ah yeah seems reasonable.

Reply:
Oh I already included this actually.